# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../heimdal/use-external-com_err.patch.cross
# Copyright (C) 2014 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

The default behavior is to always use internal libcom_err when
cross-compiling, which is not what we want when we already have
libcom_err cross-compiled and installed.

This patch changes the behavior, to always test for a working
libcom_err when a compile_et command was found in the toolchain.

--- a/cf/check-compile-et.m4	2014-03-13 19:02:24.232109566 +0100
+++ b/cf/check-compile-et.m4	2014-03-13 19:08:19.444340911 +0100
@@ -57,8 +57,6 @@
 fi
 
 if test "${krb_cv_compile_et_cross}" = yes ; then
-  krb_cv_com_err="cross"
-elif test "${krb_cv_compile_et}" = "yes"; then
   dnl Since compile_et seems to work, let's check libcom_err
   krb_cv_save_LIBS="${LIBS}"
   LIBS="${LIBS} -lcom_err"
